hyperemitter.js

```diff
@@ -58,8 +58,8 @@ function HyperEmitter (db, schema, opts) {
   })
 
   function process (change, enc, next) {
```

Comment:
I removed the call to headers directly. We don't expose .messages (.codecs) so there was no reason to use one over the other. At this point they are merged anyway.